charts/flux2/templates/pre-install-job.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.preInstallChecks -}}
```

Comment:
I think we should put the same check to https://github.com/fluxcd-community/helm-charts/blob/main/charts/flux2/templates/pre-install-job-serviceaccount.yaml

Reply:
Added in last fix